lib/generators/sorcery/install_generator.rb

```diff
@@ -90,6 +90,12 @@ def only_submodules?
         options[:migrations] || options[:only_submodules]
       end
 
+	  def rails_version # for Rails5 ActiveRecord::Migration 
+		if    Rails::VERSION::MAJOR >= 5
+		  "[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
+		end
+	  end
+
```

Comment:
This private method generate rails version( [5.0] ) for rails5 style ActiveRecord::Migration.
And nothing effect for rails3 and rails4.

Rails5 style:
`class CreateProducts < ActiveRecord::Migration[5.0]`

Rails3 and Rails4 style:
`class CreateProducts < ActiveRecord::Migration`

[Reference : RailsGuides - Active Record Migrations](http://guides.rubyonrails.org/active_record_migrations.html#migration-overview)

OK, I'll revert the spaces here too.